terraform/environments/electronic-monitoring-data/ap_airflow_iam.tf

```
@@ -613,3 +613,21 @@ module "full_reload_servicenow" {
   new_airflow = true
   full_reload = true
 }
+
+module "load_capita_blobstorage" {
+  count  = local.is-development ? 0 : 1
```

```suggestion
  count  = local.is-production || local.is-development ? 1 : 0
```
include prod and dev